hack/fetch-releases.sh

```diff
@@ -105,7 +105,11 @@ release_yaml_pac() {
     rm -rf ${dirPath} || true
     mkdir -p ${dirPath} || true
 
-    url="https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/${version}/release.yaml"
+    if [[ ${version} == "stable" ]]; then
+      url="https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/stable/release.yaml"
+    else
+      url="https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-${version}/release.yaml"
```

Comment:
For a specific version, we could use the payload attached to the release pages, like
- https://github.com/openshift-pipelines/pipelines-as-code/releases/download/0.6.1/release.yaml

The would make more sense to me than using the branch.

Reply:
either way i am fine :)